ci/backup-metadata/cleanup-cluster.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "$GKE_SERVICE_ACCOUNT_KEY" > service-account.json
+
+gcloud config set project "$(jq -r .project_id service-account.json)"
+gcloud auth activate-service-account --key-file=service-account.json
+
+cluster_name="$(cat cluster-name/info)"
+
+if gcloud container clusters describe --zone=europe-west1-b "${cluster_name}" >/dev/null 2>&1; then
+  gcloud container clusters delete --quiet --zone=europe-west1-b "${cluster_name}"
```

Comment:
If we're taking ownership of this pipeline/clusters, we should move these environments to be in `us-central1` or `us-west1` to be closer to our other environments/us

Reply:
Agreed. Given that the end goal (at least in my mind) was to make this part of https://capi.ci.cf-app.com/teams/main/pipelines/capi?group=capi-k8s-release, the need for having a `create-cluster` step may go away.